functions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * function to retrieve all cheeses as an array from db
+ * @param PDO $db
```

Comment:
line break between description/params/returns 
(please do this for all doc blocks)